src/main/java/com/massivedisaster/bintraydeployautomator/utils/GradleUtils.java

```diff
@@ -15,12 +20,34 @@
      * @param tasks            gradle tasks to execute.
      * @param arguments        arguments.
      */
-    public static void runGradle(@NotNull ProjectConnection gradleConnection, @NotNull String[] tasks, @NotNull String... arguments) {
-        gradleConnection.newBuild()
-                .forTasks(tasks)
-                .withArguments(arguments)
-                .setStandardOutput(System.out)
-                .setStandardError(System.err)
-                .run();
+    public static void runGradle(@NotNull ProjectConnection gradleConnection, @NotNull String output, @NotNull String[] tasks, @NotNull String... arguments) {
+        try {
+            OutputStream os;
+            OutputStream osErr;
+            if (output != null) {
+                File outFile = new File(output);
+                File parent = outFile.getParentFile();
+                if (parent != null) {
+                    parent.mkdirs();
+                }
+                outFile.createNewFile();
+                os = new FileOutputStream(outFile);
+                osErr = os;
+            } else {
+                os = System.out;
+                osErr = System.err;
+            }
+            gradleConnection.newBuild()
+                    .forTasks(tasks)
+                    .withArguments(arguments)
+                    .setStandardOutput(os)
+                    .setStandardError(osErr)
+                    .run();
+
+            os.close();
+            osErr.close();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
IMO, this should throw instead of catch and print.

Reply:
Fixed in next commit.